src/Makefile

```diff
@@ -154,7 +145,6 @@ full:
 	make depend
 	make tags
 	make model
-	make disagg
```

Comment:
This removed `vicDisagg` entirely as a compile-time option. Is that intentional? I assume that this can now be specified at run-time instead?